pep-9999.rst

```diff
@@ -0,0 +1,600 @@
+PEP: 9999
+Title: Support for parsing TOML in the Standard Library
+Author: Taneli Hukkinen, Shantanu Jain <hauntsaninja at gmail.com>
+Sponsor: TODO
+PEP-Delegate: TODO
+Discussions-To: TODO
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 01-Jan-2022
+Python-Version: 3.11
+Post-History: 1900-01-01
+
+
+Abstract
+========
+
+This proposes adding a module, ``tomllib``, to the standard library for
+parsing TOML. [1]_
+
+
+Motivation
+==========
+
+The TOML format is the format of choice for Python packaging, as evidenced by
+:pep:`517`, :pep:`518` and :pep:`621`. Including TOML support in the standard
+library helps avoid bootstrapping problems for Python build tools. Currently
+most Python build tools need to vendor a TOML parsing library.
+
+Python tools are increasingly configurable via TOML, for examples: ``black``,
+``mypy``, ``pytest``, ``tox``, ``pylint``, ``isort``. Those that are not, such
+as ``flake8``, cite the lack of standard library support as a `main reason why
+<https://github.com/PyCQA/flake8/issues/234#issuecomment-812800657>`_.
+
+Given the special place TOML already has in the Python ecosystem, it makes sense
+for this to be an included battery.
+
+Finally, TOML as a format is increasingly popular (some reasons for this are
+outlined in PEP 518). Hence this is likely to be a generally useful addition,
+even looking beyond the needs of Python packaging and Python tooling: various
+Python TOML libraries have about 2000 reverse dependencies on PyPI. For
+comparison, ``requests`` has about 28k reverse dependencies.
+
+
+Rationale
+=========
+
+This PEP proposes basing the standard library support for reading TOML on the
+third party library ``tomli`` [2]_.
+
+Many projects have recently switched to using ``tomli``, for example, ``pip``,
+``build``, ``pytest``, ``mypy``, ``black``, ``flit``, ``coverage``,
+``setuptools-scm``, ``cibuildwheel``.
+
+``tomli`` is actively maintained and well-tested. ``tomli`` is about 800 lines
+of code with 100% test coverage and passes all tests in a test suite `proposed as
+the official TOML compliance test suite <https://github.com/toml-lang/compliance/pull/8>`.
+
+TODO: check tomli continues to pass https://github.com/BurntSushi/toml-test
```

Comment:
```suggestion
the official TOML compliance test suite <https://github.com/toml-lang/compliance/pull/8>`,
as well as `the more established BurntSushi/toml-test suite <https://github.com/BurntSushi/toml-test>`.
```

Alright, I made sure Tomli passes up-to-date BurntSushi tests too.

BTW, I am a) not a native English speaker and b) haven't written much `rst`, so please feel free to change/fix both my language and markup as much as you want :smile: